BeeSwift/AppDelegate.swift

```diff
@@ -130,13 +130,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     }
     
     @objc func updateBadgeCount() {
-        logger.notice("Updating badge count")
```

Comment:
Some loggers are configured to include filename, function, and line number along with the logged message.
How about such a logger / format for this project?